openedx/core/lib/tests/test_request_utils.py

```diff
@@ -219,91 +219,90 @@ class CustomError2(Exception):
 
         mock_logger.info.assert_not_called()
         mock_set_custom_attribute.assert_not_called()
 
     @patch('openedx.core.lib.request_utils.set_custom_attribute')
     @patch('openedx.core.lib.request_utils.log')
     @ddt.data(None, [])
-    def test_process_exception_with_empty_expected_errors(
-        self, expected_errors_setting, mock_logger, mock_set_custom_attribute,
+    def test_process_exception_with_empty_ignored_errors(
+        self, ignored_errors_setting, mock_logger, mock_set_custom_attribute,
     ):
-        with override_settings(EXPECTED_ERRORS=expected_errors_setting):
-            ExpectedErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
+        with override_settings(IGNORED_ERRORS=ignored_errors_setting):
+            IgnoredErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
 
         mock_logger.info.assert_not_called()
         mock_set_custom_attribute.assert_not_called()
 
-    @override_settings(EXPECTED_ERRORS=[{
+    @override_settings(IGNORED_ERRORS=[{
         'MODULE_AND_CLASS': 'test.module.TestException',
-        'REASON_EXPECTED': 'Because',
+        'REASON_IGNORED': 'Because',
     }])
     @patch('openedx.core.lib.request_utils.set_custom_attribute')
     @patch('openedx.core.lib.request_utils.log')
-    def test_process_exception_not_matching_expected_errors(self, mock_logger, mock_set_custom_attribute):
-        ExpectedErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
+    def test_process_exception_not_matching_ignored_errors(self, mock_logger, mock_set_custom_attribute):
+        IgnoredErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
 
         mock_logger.info.assert_not_called()
-        mock_set_custom_attribute.assert_called_once_with('checked_error_expected_from', 'middleware')
+        mock_set_custom_attribute.assert_called_once_with('checked_error_ignored_from', 'middleware')
 
-    @override_settings(EXPECTED_ERRORS=[
+    @override_settings(IGNORED_ERRORS=[
         {
             'MODULE_AND_CLASS': 'test.module.TestException',
-            'REASON_EXPECTED': 'Because',
+            'REASON_IGNORED': 'Because',
         },
         {
             'MODULE_AND_CLASS': 'openedx.core.lib.tests.test_request_utils.CustomError1',
-            'REASON_EXPECTED': 'Because',
+            'REASON_IGNORED': 'Because',
         }
     ])
     @patch('openedx.core.lib.request_utils.set_custom_attribute')
     @patch('openedx.core.lib.request_utils.log')
-    def test_process_exception_expected_error_with_defaults(self, mock_logger, mock_set_custom_attribute):
-        ExpectedErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
+    def test_process_exception_ignored_error_with_defaults(self, mock_logger, mock_set_custom_attribute):
```

Comment:
We need to make sure this isn’t a duplicate test now.

Reply:
@robrap I did not see any duplicates and also checked in local. [u can check here](https://github.com/openedx/edx-platform/pull/33184/commits/c6a023026c3837f20045b43c5ec91b4d82573a0e)

Reply:
I'll review in my next pass, and I will mark this as resolved if I don't find anything. Thanks.